R/PlatinumGenomes-QC/Variant-Level-QC.Rmd

```diff
@@ -182,6 +182,8 @@ if(is.null(result)) {
 }
 ```
 
+See also [a version of this query](./sql/hardy-weinberg-udf.sql) that uses the new BigQuery feature of user-defined javascript functions which is currently in limited preview.  For more info, see https://www.youtube.com/watch?v=GrD7ymUPt3M#t=1377.
```

Comment:
Cloud deprecated the terms "Preview" and "Limited Preview":
 http://googlecloudplatform.blogspot.com/2014/10/new-release-phases-for-google-cloud.html

"Preview" is now "Beta" and "Limited Preview" is "Alpha".